mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1456,7 +1456,10 @@ def LLVM_LLVMFuncOp : LLVM_Op<"func", [
     OptionalAttr<BoolAttr>:$no_signed_zeros_fp_math,
     OptionalAttr<UnitAttr>:$no_inline,
     OptionalAttr<UnitAttr>:$always_inline,
-    OptionalAttr<UnitAttr>:$optimize_none
+    OptionalAttr<UnitAttr>:$optimize_none,
+    OptionalAttr<StrAttr>:$denormal_fp_math,
+    OptionalAttr<StrAttr>:$denormal_fp_math_f32,
+    OptionalAttr<StrAttr>:$fp_contract
```

Comment:
nit: Are these flags related to the other fp_math flags above? Would it make sense to group them together?

Reply:
Yeah, these flags are related to fp_math,  so i grouped these flags with other fastMath flags.